PWGLF/Tasks/spectraCharged.cxx

```diff
@@ -134,8 +139,8 @@ void chargedSpectra::init(InitContext const&)
   const int nBinsMultMeas = maxMultMeas + 1;
   const AxisSpec multMeasAxis = {nBinsMultMeas, -0.5, nBinsMultMeas - 0.5, "#it{N}^{ meas}_{ch}", "mult_meas"};
 
-  histos.add("multDist_evt_meas", "", kTH1D, {multMeasAxis});
-  histos.add("multPtSpec_trk_meas", "", kTH2D, {multMeasAxis, ptMeasAxis});
+  histos.add("multDist_evt_meas", "", kTH1D, {multMeasAxis});               //!<! measured event distribution (contains contamination from events not in specified class or with wrong vertex position)
```

Comment:
I would replace the doxygen / streamer style comments (`//!<!`) with just a simple `//`